Orange/base.py

```diff
@@ -63,15 +63,29 @@ def __call__(self, data):
         model.name = self.name
         model.original_domain = origdomain
         return model
-
+    
     def preprocess(self, data):
         """
         Apply the `preprocessors` to the data.
         """
-        for pp in self.preprocessors:
+        for pp in self.active_preprocessors:
             data = pp(data)
         return data
 
+    @property
+    def active_preprocessors(self):
+        if self.preprocessors is not None:
+            pps = []
+            if isinstance(self.preprocessors, Iterable):
+                pps += list(self.preprocessors)
+            else:
+                pps += [self.preprocessors]
```

Comment:
The thing is, it's never actually more than one preprocessor in widgets, since if we connect the `Preprocess` widget to a learner we actually get an instance of `LearnerList` or something like that. So as far as widgets go, it would make more sense to accept a single preprocessor.

As far as a programmatic API, having to wrap preprocessors in a special `PreprocessorList` whenever you want to pass in multiple preprocessors would get really annoying, really quickly.